js/app.js

```diff
@@ -1 +1,124 @@
 require('@uswds/uswds');
+
+// Add your custom javascript here
+console.log("Hi from cloud.gov Pages");
+
+// mermaid.initialize({startOnLoad:true, theme: 'base'}, ".someOtherClass");
+
+const sideNavParents = document.querySelectorAll(".usa-sidenav__item--parent-item");
+const sideNavChild = document.querySelectorAll(".usa-sidenav__sublist--middle-generation");
+const subLists = document.querySelectorAll("ul.usa-sidenav__sublist");
+const middleSubList = document.querySelectorAll(".usa-sidenav__sublist ul");
+
+// TODO: Fix issue with anchors not being included
+if (!!anchors) {
+  anchors.add('h2').remove('.no-anchor');
+  anchors.add('h3').remove('.no-anchor');
+}
+
+if (location.search === "?b") {
```

Comment:
Is this an attempt to apply a dark mode? if so, I'd rather we used CSS classes than all this JS to toggle some classes.

Reply:
Response on the other comment below.